claims_hosp/delphi_claims_hosp/update_indicator.py

```diff
@@ -96,13 +96,21 @@ def geo_reindex(self, data):
         """
         geo_map = GeoMapper()
         if self.geo == "county":
-            data_frame = geo_map.county_to_megacounty(
-                data, Config.MIN_DEN, Config.MAX_BACKWARDS_PAD_LENGTH,
-                thr_col="den", mega_col=self.geo)
+            data_frame = geo_map.fips_to_megacounty(data,
+                                                    Config.MIN_DEN,
+                                                    Config.MAX_BACKWARDS_PAD_LENGTH,
+                                                    thr_col="den",
+                                                    mega_col=self.geo)
         elif self.geo == "state":
-            data_frame = geo_map.county_to_state(data, state_id_col=self.geo)
+            data_frame = geo_map.replace_geocode(data,
+                                                 from_code="fips",
+                                                 new_col=self.geo,
+                                                 new_code="state_id")
+            data_frame[self.geo] = data_frame[self.geo]
         elif self.geo == "msa":
-            data_frame = geo_map.county_to_msa(data, msa_col=self.geo)
+            data_frame = geo_map.replace_geocode(data,
```

Comment:
I'm seeing this error when running. I don't know how the new geomap functions work, hopefully it's just a simple `new_code='X'` addition?

```
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.7/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/Cellar/python/3.7.7/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/mjahja/Delphi/covidcast-indicators/claims_hosp/delphi_claims_hosp/__main__.py", line 11, in <module>
    run_module()  # pragma: no cover
  File "/Users/mjahja/Delphi/covidcast-indicators/claims_hosp/delphi_claims_hosp/run.py", line 89, in run_module
    updater.update_indicator(params["input_file"], params["export_dir"])
  File "/Users/mjahja/Delphi/covidcast-indicators/claims_hosp/delphi_claims_hosp/update_indicator.py", line 152, in update_indicator
    data_frame = self.geo_reindex(data)
  File "/Users/mjahja/Delphi/covidcast-indicators/claims_hosp/delphi_claims_hosp/update_indicator.py", line 114, in geo_reindex
    new_col=self.geo)
TypeError: replace_geocode() missing 1 required positional argument: 'new_code'
```

Reply:
Oops meant to be `new_code` instead of `new_col`, should be fixed now.

Reply:
Works great!